src/renderers/shaders/ShaderChunk/cube_uv_reflection_fragment.glsl

```diff
@@ -1,5 +1,9 @@
+#define DISABLE_CUBE_UV_MIPMAP_INTERPOLATION
```

Comment:
It was doing interpolation between different levels of roughness (based on material roughness) as well as different mipmap levels (based on texel derivatives).  I have turned of interpolation between mipmap levels for now as I didn't see it having a huge quality impact (although we haven't tested this on complex models yet, so I worry that maybe it has an impact in complex cases, which is why I have temporarily kept it) and it did have a performance impact, at least as reported by @mattdes.

A faster way to do this would be to load the calculated roughness maps into a cubemap mipmap structure -- that option is only available in WebGL 2.0 or if the cubeMapLOD extension is available (e.g EXT_shader_texture_lod which has 70% availability.)  I think that is the next optimization -- optionally load into cubemap mipmaps when possible.  It would cut out all of this code at run time, but we would still fall back to it when the EXT_shader_texture_lod isn't available.)